app/src/main/java/com/vultisig/wallet/ui/screens/v2/home/HomePage.kt

```diff
@@ -290,6 +295,37 @@ internal fun HomePage(
     )
 }
 
+@Composable
+private fun Banners(
+    hasMigration: Boolean,
+    onMigrateClick: () -> Unit,
+    context: Context,
+    onDismissBanner: () -> Unit,
```

Comment:
is this a general component meant to be reused? if so can we inject onFollowXClick as generic callback?